### ISSUE TITLE
scripts folder not being included in zip

### ISSUE THREAD

**Issue by jamesprices** [IronCountySchoolDistrict/iep-printing#7]
The scripts folder doesn't end up in dist/plugin/web_root or in the zip file after running the gulp command. It ends up in dist/src/ instead. I have to manually put it into the zip file.

**Comment by jonesnc** [comment]
That's correct, we separate the files that need to be in the `plugin.zip` file in the `plugin` folder, and leave anything that should be copied to our image server in the `src` folder.

If you'd like to include absolutely everything in the `plugin.zip` build process, run the `build-no-img` gulp task instead. See [here](https://github.com/IronCountySchoolDistrict/gulp-ps-tasks/blob/master/index.js#L98) for more details on what's happening with that task.

**Comment by jamesprices** [comment]
Ah. That makes sense. We don't use an image server so I forgot that was a thing. I'll remember to use that task instead. That'd be good to include in the docs.
Thanks!

**Comment by jonesnc** [comment]
Do you think it would be sufficient to just mention the `build-no-img` task in the `gulp-ps-tasks` README, or should i mention in this project's README as well? I'm thinking of just modifying the `gulp-ps-tasks` README.

**Comment by jamesprices** [comment]
I think it would be good to either include it in this project's readme, or include a reference in this project's readme to the gulp-ps-tasks readme. I would have never found it without digging through the code and then googling it.
